dashboard/src/components/DeploymentFormBody/BasicDeploymentForm/Subsection.tsx

```diff
@@ -46,6 +46,7 @@ function Subsection({
           return (
             <Param
               param={childrenParam}
+              otherParams={param.children ? param.children : []}
```

Comment:
you need to modify here the component as well and include `allParams` as a property to pass it down here.

Reply:
Mmm, I think I'm not getting what you mean here. Do you mean creating the `allParams` property in the `ISubsectionProps` ?

My idea was to pass as `allParams` every children param as part of the subsection in order to make non-top-level params to work as well.
I used the example (with two levels) I wrote in https://github.com/kubeapps/kubeapps/issues/1913#issuecomment-750844297 and I worked fine.